File: Helpers/RandArrayMaker.js
module.exports = {
  returnArray,
  shuffle
};

function returnArray(num, i = 1) {
  let array = [];
  while (i <= num) {
    array.push(i);
    i++;
  }
  return array;
}

function shuffle(array) {
  for (let i = array.length - 1; i > 0; i--) {
    let x = Math.floor(Math.random() * (i + 1));
    let temp = array[x];
    array[x] = array[i];
    array[i] = temp;
  }
  return array;
}

File: Helpers/RandArrayMaker.py
from random import randint

class RandArrayMaker:
    _range = 100
    start = None
    end = None
    array = []

    def __init__(self, range):
        self.range = range

        if (range * -1) < 0:
            self.start = 1
            self.end = range
        else:
            self.start = range
            self.end = -1

        self.array = self.return_array()

    def return_array(self):
        return self.array

    def return_array(self):
        array = []
        i = self.start

        if i > self.end:
            return []

        while i <= self.end:
            array.append(i)
            i+= 1

        self.array = array
        return array

    def shuffle(self):
        if len(self.array) == 0:
            return []

        for i, num in enumerate(self.array):
            x = randint(0, len(self.array) - i)
            temp = self.array[x]
            self.array[x] = self.array[i]
            self.array[i] = temp
